Backends/PostScript/package.lisp

```diff
@@ -19,7 +19,7 @@
 ;;; Boston, MA  02111-1307  USA.
 
 (cl:defpackage #:clim-postscript
-  (:use #:clim #:clim-extensions #:clim-lisp #:clim-postscript-font)
+  (:use #:clim #:clime #:climb #:clim-lisp #:clim-postscript-font)
```

Comment:
Can you write `climb:find-port-type` instead of importing this? In my experience, `use`ing to many packages in backends has already let to lots of confusion.

Reply:
I can, however I've put it in use because clim-postscript is the only exception among backends that do not use climb, so I thought that consistency will be good here.

Reply:
I would like to move into the opposite direction such that eventually no backend `use`s `climb`, but you decide.

Reply:
in that case I'm closing this